assets/js/romo/base.js

```diff
@@ -881,6 +860,27 @@ Romo.prototype._elemsWrapMap = {
   'td':       [3, "<table><tbody><tr>", "</tr></tbody></table>"]
 };
 
+Romo.prototype._elemsInitComponents = {};
+
+Romo.prototype._elemsInitTrigger = function(onElems) {
+  for (var selector in this._elemsInitComponents) {
+    this._elemsInitFind(onElems, selector).forEach(Romo.proxy(function(initElem) {
+      new this._elemsInitComponents[selector](initElem);
```

Comment:
@kellyredding - I think you could pull out `this._elemsInitComponents[selector]` in a local var and use it in this function. This would allow dropping the `Romo.proxy` and avoiding the lookup in the loop. Not sure it's more efficient but maybe.